backend/src/express.ts

```diff
@@ -0,0 +1,27 @@
+import express, {Request, Response } from 'express';
+import cookieParser from 'cookie-parser';
+import compression from 'compression';
+import morgan from 'morgan';
+import cors from 'cors';
+import apiRouter from './routes/api.router';
+// import healthRoute from './routes/health';
+
+const app = express();
+
+// Express middleware
+app.use(express.json());
+app.use(express.urlencoded({ extended: false }));
+// app.use(cookieParser());
+// app.use(compression());
+// app.use(morgan('dev'));
```

Comment:
This should be initialized or removed. Adding commented out code just leaves more tech debt for your future self. You've already imported these tools too, so you're leaving behind unused variables that a linter would flag